components/gardener/virtual/deployment.yaml

```diff
@@ -69,6 +69,8 @@ dns_credentials:
     OS_DOMAIN_NAME: (( base64( landscape.dns.credentials.domainName ) ))
     OS_USER_DOMAIN_NAME: (( base64( landscape.dns.credentials.userDomainName || "" ) ))
     OS_PROJECT_NAME: (( base64( landscape.dns.credentials.tenantName ) ))
+  cloudflare-dns:
+    CLOUDFLARE_API_TOKEN: (( base64( landscape.dns.credentials.apiToken ) ))
```

Comment:
This transformation of credentials is legacy code, from a time where the Gardener and the dns-controller-manager needed different keys for the credentials. I removed a similar mapping from the DNS controller component (see https://github.com/gardener/garden-setup/commit/8d14a033e8dbdab01c3c440304023e7c6fde2f7a), but I seem to have missed this one. 

I already created a PR to remove this mapping, I will merge it into master soon, so you can also remove this part (see https://github.com/gardener/garden-setup/pull/212).

Reply:
Reverted this change